app/assets/stylesheets/core/margins.scss

```diff
@@ -79,7 +79,7 @@
 }
 
 .smallMarginBottom {
-  margin-bottom: $size-20;
+  margin-bottom: $size-14;
```

Comment:
Let's not change this value since this is a global class that's used everywhere in the application.

Reply:
Acknowledged.